apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -47,6 +36,44 @@ Using [PublisherServiceApiClient](obj/api/Google.Cloud.PubSub.V1.PublisherServic
 
 {{sample:SubscriberServiceApiClient.Overview}}
 
+# Performance considerations and default settings
+
+`PublisherClient` and `SubscriberClient` are optimized for high-throughput high-performance scenarios,
+and default settings have been chosen with this in mind;
+however, note that these classes are also well suited to use cases where performance is not a major
+consideration.
+
+By default multiple gRPC channels are created on client startup, with the channel count defaulting to
+the CPU processor count as returned by `Environment.ProcessorCount`. This is to allow greater bandwidth
+than a single gRPC channel can support; the processor count is is a pragmatic choice to approximately
+scale maximum throughput performance by potential machine workload.
+
+If using multiple clients on a machine with a high processor count, then this may cause problems
```

Comment:
```suggestion
When using multiple clients on a machine with a high processor count, this may cause problems
```

("If using" doesn't sound quite right to me. "If you are using" would be okay, as an alternative)

Reply:
Done (not using your suggestion due to CLA issue)